build-system/eslint-rules/no-non-string-log-args.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @typedef {{
+ *   name: string.
+ *   variadle: boolean,
+ *   startPos: number
+ * }}
+ */
+const LogMethodMetadata = null;
```

Comment:
`let LogMethodMetadataDef`

Reply:
I'm not seeing a `reassignment` here but agree with the name change so it's more clear this is a type reference.

Reply:
You can't have `const` without an initial assignment. The pattern in the rest of the codebase is to use `let`.

Reply:
done.